layouts/partials/header.html

```diff
@@ -1,7 +1,11 @@
-<div id="header">
-    <h1><a href="{{ .Site.BaseURL }}" id="logo">
-        {{ with .Site.Data.globalheader.title }}{{ . | markdownify }}{{ else }}{{ .Site.Title }}{{ end }}
-    </a></h1>
+{{ $globalLogoEnabled := or (not (isset .Site.Data.globalheader.logo "enable")) .Site.Data.globalheader.logo.enable }}
```

Comment:
If `.Site.Data.globalheader.logo` isn't defined, the `isset` will cause a warning

Reply:
Fixed with new commit. Now checks that `.Site.Data.globalheader` and  `.Site.Data.globalheader.logo` are defined.